### PR TITLE
Add prize section

### PR DESCRIPTION
**Pull request by uy-seng**
TODO: just need to add prize and challenge info to firebase

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -107,6 +110,18 @@ export default function Home(props: {
...
+  //   {},
+  // );
+  const { data: challengeData } = await RequestHelper.get<Challenge[]>(
+    `${protocol}://${context.req.headers.host}/api/challenges/`,
+    {},
+  );
+  const { data: prizeData } = await RequestHelper.get<Array<{ rank: number; prizeName: string }>>(
+    `${protocol}://${context.req.headers.host}/api/prizes`,
+    {},
+  );
```
Comment (DedsecKnight) [inline pages/index.tsx:124]: I don't think we actually need this?

File: pages/index.tsx [hunk 2]
```diff
@@ -121,9 +136,12 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
   );
   return {
     props: {
+      // keynoteSpeakers: keynoteData,
+      challenges: challengeData,
       answeredQuestion: answeredQuestion,
       scheduleCard: scheduleData,
       dateCard: dateData,
+      prizeData: prizeData,
```
Comment (DedsecKnight) [inline pages/index.tsx:144]: ^

**Review by DedsecKnight — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (uy-seng) [reply pages/index.tsx:124]: where are we pulling the prize data from?

**Review by DedsecKnight — COMMENTED** [review]
lgtm :+1:

**Pull request merged**